packages/help/__tests__/__integration__/__snapshots__/BrightHelpTests.integration.test.ts.snap

```diff
@@ -25,7 +25,7 @@ exports[`Root level help tests top level help should contain support link 1`] =
  GROUPS
  ------
 
-   auth                  Authentication commands                          
+   auth                  Manage tokens for authentication services        
```

Comment:
General comment:
Have we had Doc and/or UX review the help text overall?
Or should we rather try reviewing everything once the token branches PRs are opened?

Reply:
I don't think they have. That is something we are going to want them to do ASAP, probably before we approve this.

Reply:
Deferring doc and UX review to another PR once merged.